types/framework/entities/actor.d.ts

```diff
@@ -293,48 +293,63 @@ declare class Actor<
 
   /** @override */
   createEmbeddedEntity(
-    embeddedName: string,
-    data: Optional<ActiveEffect | Actor.OwnedItem<I>>,
+    embeddedName: 'OwnedItem',
+    data: Optional<Actor.OwnedItem<I>>,
     options?: any
-  ): Promise<ActiveEffect | Actor.OwnedItem<I>>;
+  ): Promise<Actor.OwnedItem<I>>;
+
```

Comment:
Are overloads not documented the same anyway when you put them together? We could just remove the doc comment and empty line in that case.

Reply:
done